src/dns/coredns.rs

```diff
@@ -222,39 +242,24 @@ impl CoreDns {
                                     reply(sender.clone(), src_address, &nx_message).unwrap();
                                 } else {
+                                    let nameservers = self.resolv_conf.nameservers.clone();
                                     tokio::spawn(async move {
-
                                         // forward dns request to hosts's /etc/resolv.conf
-                                        if let Ok(resolver) = TokioAsyncResolver::tokio_from_system_conf() {
-                                            if let Ok(lookup_ip) = resolver.lookup_ip(name.as_str()).await {
-                                            let record_name: Name = Name::from_str_relaxed(name.as_str()).unwrap();
-                                                for response_ip in lookup_ip {
-                                                        match response_ip {
-                                                            IpAddr::V4(ipv4) => {
-                                                                req.add_answer(
-                                                                     Record::new()
-                                                                        .set_name(record_name.clone())
-                                                                        .set_ttl(86400)
-                                                                        .set_rr_type(RecordType::A)
-                                                                        .set_dns_class(DNSClass::IN)
-                                                                        .set_rdata(RData::A(ipv4))
-                                                                        .clone(),
-                                                                );
-                                                            },
-                                                            IpAddr::V6(ipv6) => {
-                                                                req.add_answer(
-                                                                     Record::new()
-                                                                        .set_name(record_name.clone())
-                                                                        .set_ttl(86400)
-                                                                        .set_rr_type(RecordType::AAAA)
-                                                                        .set_dns_class(DNSClass::IN)
-                                                                        .set_rdata(RData::AAAA(ipv6))
-                                                                        .clone(),
-                                                                );
-                                                            }
-                                                        }
+                                        for nameserver in nameservers {
```

Comment:
If all servers fail, do we respond at all?

Reply:
@mheon It should time out. It was decided by everyone in the last PR.

Reply:
Alright, seems reasonable.